src/functions-and-arrays.js

```diff
@@ -1,41 +1,141 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  } else if (number2 > number1) {
+    return number2;
+  } else if (number1 === number2) {
+    return number1;
+  }
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arrayWords) {
+  const numOfLetters = [];
+  let indexOfMax;
+  let maxNumOfLetters;
+
+  if (arrayWords.length === 0) {
+    return null;
```

Comment:
just a recommendation, it would be even easier to read your code if you check array's validity first. And only then start another if statement:

```
if (arrayWords.length === 0) return null;

if (arrayWords.length === 1) {
  // remaining logic here
}
```

If you write it like this, you very clearly separate the logic. If array s not valid i just exit the function. If valid - i carry on. Also if you noticed, if my return is only one line I do not need curly braces!

And even more advanced syntax could be this:
`if (!arrayWords.length) return null;`

Here, if `arrayWords.length = 0` it will return false (in js zero is falsy), so we negate it with explamation point to get desired result.